account_credit_control_dunning_fees/tests/test_fees_generation.py

```diff
@@ -16,7 +16,18 @@ def setUp(self):
         self.assertTrue(self.usd)
 
         self.company = self.browse_ref('base.main_company')
-        self.company.currency_id = self.euro
+        # travis fails
+        if self.company.currency_id != self.euro:
+            self.env.cr.execute("""
+                UPDATE res_company
+                SET currency_id = %(currency)s
+                WHERE id = %(company)s
+                """, {'currency': self.euro.id,
+                      'company': self.company_id.id})
+            # invalidate then reload cache after sql update
+            self.env.invalidate(self.env['res.company']._fields['currency_id'],
+                                self.company.ids)
+            self.company = self.browse_ref('base.main_company')
```

Comment:
This should not be necessary. Because of the invalidate() call, whenever self.company.currency_id is referenced, it will be reread from the database.